nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/java/org/apache/nifi/lookup/RestLookupService.java

```diff
@@ -214,6 +226,7 @@ protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
     private volatile String basicUser;
     private volatile String basicPass;
     private volatile boolean isDigest;
+    private volatile PropertyValue responseHandler;
```

Comment:
This can be changed to use the `ResponseHandlingStrategy` enum:
```suggestion
    private volatile ResponseHandlingStrategy responseHandlingStrategy;
```